src/main.rs

```diff
@@ -52,6 +54,18 @@ async fn main() -> anyhow::Result<()> {
         ));
     }
 
+    match config.anti_hoisting {
+        Some(anti_hoisting_config) if anti_hoisting_config.enabled => {
+            tokio::spawn(AntiHoisting::serve(
+                anti_hoisting_config,
+                event_rx.resubscribe(),
+                http.clone(),
+            ));
+        }
+        Some(_) => warn!("Anti-hoisting is disabled in the config; not enabling anti-hoisting"),
```

Comment:
I feel like it is kind of redundant to both make the section option and include a 'enabled' field. IMO it should just be enabled if the configuration is present.